coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/MultiThreadedEventProcessor.java

```diff
@@ -90,6 +90,7 @@ public MultiThreadedEventProcessor(
      * @param time              The time.
      * @param eventAccumulator  The event accumulator.
      */
+    @SuppressWarnings("this-escape")
     public MultiThreadedEventProcessor(
```

Comment:
This class is not used in mocking, so could you please add `final` to the class instead of suppress?

see #17409 for example

Reply:
Done @chia7712 .